script/hassfest/requirements.py

```diff
@@ -30,7 +30,6 @@
 
 IGNORE_STANDARD_LIBRARY_VIOLATIONS = {
     # Integrations which have standard library requirements.
-    "slide",
     "suez_water",
```

Comment:
Oh, we recently changed libraries for suez_water, so I think we can get rid of this list.

Reply:
> Oh, we recently changed libraries for suez_water, so I think we can get rid of this list.

Probably not. Otherwise `hassfest` would have failed.

Reply:
@jb101010-2 Can we drop the `asyncio` dependency from pyseuzv2?

Reply:
I see... It's a bit more complicated as it first seemed. `pysuezV2` version `0.2.1` (the current one used by HA) depends on `datetime`. That dependency was removed in `0.3.0` but it also added a new one on `asyncio`.
https://github.com/jb101010-2/pySuez/commit/ba58a6832aaf41efb742dd59fdd92dd58d74eae4

I.e. if we go ahead an merge this, we'd have to readd `asyncio` as exception to `licenses.py` when we update `pysuezV2` before the dependency is removed.

Probably still fine though.

Reply:
In that case we just don't merge until it is fixed ;)

Reply:
> In that case we just don't merge until it is fixed ;)

Opened https://github.com/jb101010-2/pySuez/pull/4 upstream. Is that enough for this one ;) ?

Reply:
> @jb101010-2 Can we drop the `asyncio` dependency from pyseuzv2?

I'm new to python so if you all say it's ok without it I don't see why not 😅

> > In that case we just don't merge until it is fixed ;)
> 
> Opened [jb101010-2/pySuez#4](https://github.com/jb101010-2/pySuez/pull/4) upstream. Is that enough for this one ;) ?

I'll check during the week-end and I'll get back with an update.

Reply:
I made a [PR](https://github.com/home-assistant/core/pull/129205) removing datetime.  
Asyncio was not used for now, I'll make sure it's not added in the future release I'm working on